data-raw/generate-vars.R

```diff
@@ -0,0 +1,89 @@
+# This script is used to update the string-encoded tree and
+# associated/mapped time-zones from https://raw.githubusercontent.com/darkskyapp/tz-lookup/master/tz.js
+# that are stored in src/generated-vars.h
+
+library(stringi)
+library(Rcpp)
+
+tf <- tempfile(fileext = ".js")
+on.exit(unlink(tf), add=TRUE)
+
+# Grab the pre-computed string tree and mapped TZ list
+download.file(
+  "https://raw.githubusercontent.com/darkskyapp/tz-lookup/master/tz.js",
```

Comment:
This is downloading the latest `tz.js`, but the "accurate" method is currently using the [`2018d` version from time-zone-boundary-builder](https://github.com/evansiroky/timezone-boundary-builder/releases/tag/2018d), so I think that is probably the source of some of the differences when comparing "fast" and "accurate".

I tried quickly with (what I think is) [the analogous `tz.js`](https://raw.githubusercontent.com/darkskyapp/tz-lookup/01136c2e0de64ea5380c1f71aedd5cec351ba348/tz.js), but it looks like the array is constructed slightly differently so the regex-wrangling didn't quite work... I'm happy to look at this further, as I think it would be good to use the same version in "fast" and "accurate"

Reply:
This was sorted out by moving both methods to use the latest (2019b) boundary in #2.